docs/reference/docs/get.asciidoc

```diff
@@ -109,11 +109,11 @@ PUT twitter
          "properties": {
             "counter": {
                "type": "integer",
-               "store": "no"
+               "store": "false"
             },
             "tags": {
                "type": "keyword",
-               "store": "yes"
+               "store": "true"
```

Comment:
I'm fairly sure you don't need the quotes here and it'd be more idiomatic not to have them.